cli/ballerina-launcher/src/main/java/org/ballerinalang/launcher/Main.java

```diff
@@ -186,42 +177,48 @@ private static String prepareCompilerErrorMessage(String message) {
         return "ballerina: " + LauncherUtils.makeFirstLetterLowerCase(message);
     }
 
+    private static String getFirstUnknownArg(String errorMessage) {
+        String optionsString = errorMessage.split(":")[1];
+        return (optionsString.split(","))[0].trim();
+    }
+
     /**
      * This class represents the "run" command and it holds arguments and flags specified by the user.
      *
      * @since 0.8.0
      */
-    @Parameters(commandNames = "run", commandDescription = "compile and run Ballerina program")
+    @CommandLine.Command(name = "run", description = "Compiles and runs Ballerina programs")
```

Comment:
We have changed starting from lower case to upper case on the description test. I thought we followed lower case for command description that gets printed on the console.

Reply:
Also the wording has to be from third person context. For example check man page for ls unix command

```
LS(1)                     BSD General Commands Manual                    LS(1)

NAME
     ls -- list directory contents

```

Reply:
Shall we follow that approach for all the commands ?